FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo.xcodeproj/project.pbxproj

```diff
@@ -80,7 +81,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				0C73704BAABBD042032D31D6 /* BuildFile in Frameworks */,
+				0C73704BAABBD042032D31D6 /* (null) in Frameworks */,
```

Comment:
Huh.

Reply:
Yeah I know eh? I tried to figure out what the heck was generating this but I couldn't. No setting seemed to map to this.